helm-chart/values.yaml

```diff
@@ -0,0 +1,108 @@
+---
+releaseImageTag: cf66cfb # Need to pass this value when applying the helm chart (This is the $BUILD_ID )
+sandbox: "false"
+sandboxName: ""
+deployEnv: "deadbeef" # This value should not appear in finalized templates # Need to pass this value when applying the helm chart
+contentSandbox: "false"
+previousServicesCount: "3"
+projectID: development-205018 # Need to pass this value when applying the helm chart
+
+web:
+  containerImage: 
+    repoLocation: gcr.io
+    repoName: sefaria-web
+  replicaCount: 7
+  resources:
+    web:   
+      gunicornWorkerCount: 5
+      gunicornThreadCount: 5
+      request:
+        memory: "10Gi"
+        cpu: "1500m"
+      limit:
+        memory: "11Gi"
+        cpu: "2000m"
+    djangoLog:
+      request:
+        memory: "100Mi"
+        cpu: "50m"
+      limit:
+        memory: "150Mi"
+        cpu: "100m"
+
+redis:
+  resources:
+    request:
+      memory: "700Mi"
+      cpu: "200m"
+    limit:
+      memory: "700Mi"
+      cpu: "200m"
+
+nodejs:
+  containerImage: 
+    repoLocation: gcr.io
+    RepoName: sefaria-node
+  replicaCount: 4
+  resources:
+    request:
+      memory: "1Gi"
+      cpu: "1000m"
+    limit:
+      memory: "1500Mi"
+      cpu: "1500m"
+
+varnish:
+  containerImage:
```

Comment:
I would change this a string of the full container image repo/path/version - The calculation of the image tag should be external to the helm chart.

Reply:
Separate repo/path/image fields are fairly standard though, since it lets a consumer of the chart point at a mirrored repo while still consuming bumps to default image numbers through releases.

Reply:
Perhaps  - but then there is no need for a separate repoLocation and repoName it would should be container.image and container.version